govCMS/govcms8lagoon
# Feature/govcms 615 solr configuration

This PR adds support for Lagoon Solr module to create a Solr Server.
Steps to QA/test:

1. Build environment `ahoy build`
2. Enable `lagoon_search` module: `ahoy drush en lagoon_search`
3. Login to the site `ahoy login` and navigate to http://govcms8-lagoon.docker.amazee.io/admin/config/search/search-api
4. Make sure the **Lagoon Solr** server is available in the table and shows green status.
